src/features/subscription/NotificationsConnectionModal.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { AuthenticationButton } from 'features/auth/components/AuthenticationButton/AuthenticationButton'
+import { StepperOrigin } from 'features/navigation/RootNavigator/types'
+import { analytics } from 'libs/analytics'
+import { ButtonWithLinearGradient } from 'ui/components/buttons/buttonWithLinearGradient/ButtonWithLinearGradient'
+import { AppModalWithIllustration } from 'ui/components/modals/AppModalWithIllustration'
+import { InternalTouchableLink } from 'ui/components/touchableLink/InternalTouchableLink'
+import { BicolorUserNotification } from 'ui/svg/icons/BicolorUserNotification'
+import { Spacer, Typo } from 'ui/theme'
+
+interface Props {
+  visible: boolean
+  dismissModal: () => void
+}
+
+export const NotificationsConnectionModal: FunctionComponent<Props> = ({
```

Comment:
Dans l'app `connection` veut souvent dire que l'utilisateur est connecté à Internet.
```suggestion
export const NotificationsLoggedOutModal: FunctionComponent<Props> = ({
```